app/src/db/migrations/20220130133615_000-init.js

```diff
@@ -0,0 +1,129 @@
+const stamps = require('../stamps');
+const uuid = require('uuid');
+
+const CREATED_BY = uuid.NIL;
+
+exports.up = function (knex) {
+  return Promise.resolve()
+
+    // Create user table and add a dummy system user (to link migration CreatedBy stamps)
+    .then(() => knex.schema.createTable('oidc_user', table => {
+      table.string('oidcId').primary();
```

Comment:
do we want to use oidc in the label.. would we ever want to authenticate using tokens from another source (eg webade).
probably not, right